### PR TITLE
Added table sample code

### LINKED ISSUE #116: Create a Table Admin Sample
There needs to be a sample in the samples/ directory that manipulates tables. Here's the list of commands:

- **createTable [projectId] [instanceId]**
    - Creates a table named "my_table" with 1 column family "simpleFamily" that has a max versions of 1
- **checkTableExistence  [projectId] [instanceId]**
    - Does a list tables, and displays true if the table is in the list.
- **getTable [projectId] [instanceId]**
    - Gets "my_table", and displays the column families and their gcPolicy
- **listTables  [projectId] [instanceId]**
    - // lists all of the tables
- **deleteTable [projectId] [instanceId]**
   - Deletes "my_table"
- **addComplexColumnFamily [projectId] [instanceId]**
   - Creates a column family called "complex" on "my_table" that has the following GCRule which means "have at least 2 columns, but no more than 10.  Any additional columns past 2 should have a TTL of 30d: 
```
   union: [
       {max_versions: 10}
       {intersection: [
         {max_versions: 2}
         {max_age: 30d}
       ]}
   ]
```
- **deleteComplexColumnFamily [projectId] [instanceId]**
   - Deletes the "complex" column family

### PR DESCRIPTION
**Pull request by vijay-qlogic**
## Do not merge: [Waiting to determine the sample reference names](https://github.com/googleapis/nodejs-bigtable/pull/131#issuecomment-388067801)

Fixes #116  (it's a good idea to open an issue first for discussion)

- [ ] Tests and linter pass
- [ ] Code coverage does not decrease (if any source code was changed)
- [ ] Appropriate docs were updated (if necessary)

### CODE REVIEW

File: samples/tables.js [hunk 1]
```diff
@@ -0,0 +1,279 @@
+/**
+ * Copyright 2017, Google, Inc.
```
Comment (stephenplusplus) [inline samples/tables.js]: 2018!

**Comment by stephenplusplus** [comment]
Do these samples need the typical `[sample_start]` brackets?

**Comment by sduskis** [comment]
@stephenplusplus: The samples need the typical brackets.  We will have to work with other folks to make that happen.  This PR should not be merged until we work things out.

File: samples/tables.js [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+  // [START create_column_family]
+  // refer table with id 'my-table'
+  const table = instance.table('my-table');
+
+  const options = {
+    rule: {
+      age: {
+        seconds: 60 * 60 * 24 * 30,
+      },
+      versions: 2,
```
Comment (jabubake) [inline samples/tables.js]: @sduskis @vijay-qlogic  How is "no more than 10" supported in this rule ?

**Comment by sduskis** [comment]
I'm going to close this.  #139 implements table management.

**Pull request closed without merging**